src/zenml/integrations/seldon/seldon_client.py

```diff
@@ -0,0 +1,735 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
```

Comment:
I believe all the classes within this file could benefit from descriptive docstring that helps understand what the class does/represents. The docstrings that some of the classes already have are perfect :)